v2/common/pom.xml

```diff
@@ -28,7 +28,7 @@
 
     <properties>
         <avro.version>1.8.2</avro.version>
-        <bigtable.version>1.15.0</bigtable.version>
+        <bigtable.version>2.6.3</bigtable.version>
```

Comment:
can we rename this to bigtable-hbase.version?

Reply:
Done. I didn't change the naming convention under syndeo-template/pom.xml since it's unrelated. I can open a separate PR if we want to change it in the main repo.